UGW: define a custom sub-target image path to copy final images based on profile name or subtarget name

diff --git a/include/target.mk b/include/target.mk
--- a/include/target.mk
+++ b/include/target.mk
@@ -34,6 +34,7 @@ endif
 
 TARGETID:=$(BOARD)$(if $(SUBTARGET),/$(SUBTARGET))
 PLATFORM_SUBDIR:=$(PLATFORM_DIR)$(if $(SUBTARGET),/$(SUBTARGET))
+TARGET_IMG_DIR:=$(BIN_DIR)/$(if $(BUILD_SUFFIX),$(BUILD_SUFFIX),$(if $(SUBTARGET),$(SUBTARGET)))
 
 ifneq ($(TARGET_BUILD),1)
   ifndef DUMP
